src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/StringLiteralEqualityCheck.java

```diff
@@ -46,7 +46,20 @@
  * <pre>
  * &lt;module name=&quot;StringLiteralEquality&quot;/&gt;
  * </pre>
+ * <p>
+ * Examples of violations:
+ * </p>
+ * <pre>
+ * if (status == "done")            // violation
```

Comment:
please make this and other comments like `if (status == "done") // violation`, with only one white space separation

Reply:
@strkkk Applied all the changes, let me know if there's anything else to be changed